website/src/components/canvas/Sidebar.jsx

```diff
@@ -104,16 +138,34 @@ class Sidebar extends React.Component {
       <div
       onMouseEnter={this.mouseEnter}
       onMouseLeave={this.mouseLeave}>
-
+       <IconButton
+        iconStyle={styles.arrowOpen}
+        tooltip="Show sidebar"
+        tooltipPosition="bottom-right"
+        onClick={this.onClickOpen}
+        style={{opacity: this.state.closed}}>
+        {this.state.closed ? 0 : 1}
+        <ArrowForward/>
+      </IconButton>
         <Drawer containerStyle={styles.sidebar}
-                open= {true}
+                open= {this.state.isOpen}
                 docked={true}
                 openSecondary={false}
                 style={{opacity: this.state.opacity}}
                 zDepth={0}>
           <div
           style={styles.propertiesSpacing}>
           <ul>
```

Comment:
"that is generally the standard for drawer sidebars that are collapsible." can you elaborate on this? Besides the hamburger menu, I can't find any other design standards for using arrows?